src/main/java/dbService/DBService.java

```diff
@@ -0,0 +1,95 @@
+package dbService;
+
+import com.mysql.jdbc.Driver;
+import dbService.dao.UsersDAO;
+import dbService.dataSets.UsersDataSet;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBService {
+
+    private static String URL = "jdbc:mysql://localhost:3306/clients?";
+    private static String LOGIN = "user=root&";
+    private static String PASSWORD = "password=1111";
+    private final Connection connection;
+
+    public DBService() {
+        this.connection = getMysqlConnection();
+    }
+
+    public UsersDataSet getUser(String login) throws DBException {
+        try {
+            return (new UsersDAO(connection).get(login));
+        } catch (SQLException e) {
+            throw new DBException(e);
+        }
+    }
+
+    public long addUser(String login, String password, String email) throws DBException {
+        try {
+            connection.setAutoCommit(false);
+            UsersDAO dao = new UsersDAO(connection);
+            dao.createTable();
+            dao.insertUser(login, password, email);
+            connection.commit();
+            return dao.getUserId(login);
+        } catch (SQLException e) {
+            try {
+                connection.rollback();
+            } catch (SQLException ignore) {
+            }
+            throw new DBException(e);
+        } finally {
+            try {
+                connection.setAutoCommit(true);
+            } catch (SQLException ignore) {
+            }
+        }
+    }
+
+    public void cleanUp() throws DBException {
+        UsersDAO dao = new UsersDAO(connection);
+        try {
+            dao.dropTable();
+        } catch (SQLException e) {
+            throw new DBException(e);
+        }
+    }
+
+    public void printConnectInfo() {
+        try {
+            System.out.println("DB name: " + connection.getMetaData().getDatabaseProductName());
+            System.out.println("DB version: " +
+                    connection
+                            .getMetaData()
+                            .getDatabaseProductVersion());
+            System.out.println("Driver: " + connection.getMetaData().getDriverName());
+            System.out.println("Autocommit: " + connection.getAutoCommit());
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static Connection getMysqlConnection() {
+        try {
+            DriverManager
+                    .registerDriver((Driver) Class.forName("com.mysql.jdbc.Driver")
+                            .newInstance());
+
+            StringBuilder fullUrl = new StringBuilder();
+
+            fullUrl.
+                    append(URL).
+                    append(LOGIN).
+                    append(PASSWORD);
+
+            return DriverManager.getConnection(fullUrl.toString());
```

Comment:
maybe better write:
```suggestion
            return DriverManager.getConnection(URL + LOGIN + PASSWORD);
```
and then remove builder?